tests/unit/extensions/test_gunicorn.py

```diff
@@ -127,6 +129,85 @@ def test_flask_extension_default(tmp_path, flask_input_yaml, packages):
     }
 
 
+@pytest.mark.parametrize(
+    "files,organize,command",
+    [
+        (
+            {"app.py": "app = object()"},
+            {"app.py": "flask/app/app.py"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app",
+        ),
+        (
+            {"app/__init__.py": "app = object()"},
+            {"app": "flask/app/app"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app",
+        ),
+        (
+            {"app/__init__.py": "from .app import app"},
+            {"app": "flask/app/app"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app",
+        ),
+        (
+            {"app/app.py": "app = object()"},
+            {"app": "flask/app/app"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app.app:app",
+        ),
+        (
+            {"app/main.py": "app = object()"},
+            {"app": "flask/app/app"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app.main:app",
+        ),
+        (
+            {"app/app.py": "app = object()", "app/__init__.py": "from .app import app"},
+            {"app": "flask/app/app"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app",
+        ),
+        (
+            {"src/app.py": "app = object()"},
+            {"src": "flask/app/src"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py src.app:app",
+        ),
+        (
+            {"foo_bar/app.py": "app = object()"},
+            {"foo_bar": "flask/app/foo_bar"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py foo_bar.app:app",
+        ),
+        (
+            {"app.py": "app = object()", "foo_bar/app.py": "app = object()"},
+            {"app.py": "flask/app/app.py"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app",
+        ),
+        pytest.param(
+            {"main.py": "app = object()", "src/app.py": "app = object()"},
+            {"main.py": "flask/app/main.py"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py main:app",
+            id="With two entrypoints, take the first one",
+        ),
+        pytest.param(
+            {"src/app.py": "app = object()", "migrate.sh": "", "unknown": ""},
+            {"src": "flask/app/src", "migrate.sh": "flask/app/migrate.sh"},
+            "/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py src.app:app",
+            id="Include other files beside the entrypoint",
+        ),
```

Comment:
Can you also add unit tests for the other use-cases like `src/main.py`, `src/__init__.py`, `foo_bar/main.py` and `foo_bar/__init__.py`?